android/src/main/java/com/appcuesreactnative/ReactNativeElementTargeting.kt

```diff
@@ -146,10 +146,10 @@ private fun View.asCaptureView(screenBounds: Rect): ViewElement? {
 
 internal fun View.selector(): ReactNativeViewSelector? {
     return ReactNativeViewSelector(
+        // captures "nativeID" property set on a react native View
         nativeId = getTag(R.id.view_tag_native_id)?.toString(),
-        // a "testID" set on a react native view will come in to the Android View in the
-        // tag property
-        testId = tag as? String
+        // captures "testID" property set on a react native View
+        testId = getTag(R.id.react_test_id)?.toString(),
```

Comment:
using react_test_id here (similar to the native ID usage above) is the "more correct" way to get the exact tag value expected.

here is where it is set in react native https://github.com/facebook/react-native/blob/5ba8de05b5e6c170b84b44979e7bbc576442eafb/packages/react-native/ReactAndroid/src/main/java/com/facebook/react/uimanager/BaseViewManager.java#L192